apps/hbl365/src/Main.purs

```diff
@@ -0,0 +1,165 @@
+module HBL365.Main where
+
+import Prelude
+
+import Data.Array as Array
+import Data.Either (Either)
+import Data.Maybe (Maybe(..))
+import Data.Set as Set
+import Effect.Aff as Aff
+import Effect.Class (liftEffect)
+import Effect.Exception (Error)
+import Effect.Unsafe (unsafePerformEffect)
+import HBL365.Product (getProduct)
+import HBL365.NewPurchase as NewPurchase
+import KSF.Spinner as Spinner
+import KSF.User as User
+import KSF.Vetrina as Vetrina
+import React.Basic (JSX)
+import React.Basic.DOM as DOM
+import React.Basic.Hooks (Component, component, useEffectOnce, useState', (/\))
+import React.Basic.Hooks as React
+import Vetrina.Types (Product)
+
+foreign import appStore ::
+  { apple   :: String
+  , android :: String
+  }
+
+jsApp :: {} -> JSX
+jsApp = unsafePerformEffect app
+
+app :: Component {}
+app = do
+  component "HBL365" \_ -> React.do
+    product /\ setProduct <- useState' Nothing
+    useEffectOnce do
+      Aff.launchAff_ do
+        liftEffect <<< setProduct <<< Just <<< map Array.singleton =<< getProduct
+      pure $ pure unit
+    pure $ render product
+
+render :: Maybe (Either Error (Array Product)) -> JSX
+render product =
+  React.fragment
+    [ DOM.header_
+        [ DOM.div
+            { className: "container"
+            , children:
+                [ DOM.a
+                    { id: "logo-container"
+                    , href: "/"
+                    , children:
+                        [ DOM.img
+                            { src: "https://customer.hbl.fi/images/new/app-icon-1024x1024-pixels.png" }
```

Comment:
We should probably include this image in the repo directly

Reply:
In fact I did but this URL was still left to point to the old location. Will fix.